commands/faas_test.go

```diff
@@ -0,0 +1,53 @@
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"testing"
+)
+
+func TestLoadsDefaultYAMLWhenPresent(t *testing.T) {
+	rs := resetState()
+	defer rs()
+	os.Chdir("./test_fixtures")
+
+	Execute([]string{"help"})
+
+	if yamlFile != "stack.yml" {
+		t.Fatal(fmt.Printf("Expected yamlFile to equal %v got %v\n", "stack.yml", yamlFile))
+	}
+}
+
+func TestLoadsFromPametersYAMLWhenGivenAndDefaultPresent(t *testing.T) {
+	rs := resetState()
+	defer rs()
+	os.Chdir("./test_fixtures")
```

Comment:
Would you be okay with changing the name of the directory to `testdata`? We have a [PR out](https://github.com/openfaas/faas-cli/pull/87) that creates this directory. Also, the go source code mentions that `testdata` dirs are [ignored by the compiler](https://go-review.googlesource.com/c/go/+/23353/1/src/cmd/go/test.go#62). Let me know your thoughts. Thanks!